src/assets/scss/theme/_icons.scss

```diff
@@ -1199,4 +1199,19 @@ $font-path: "/bda-des-sys/assets/fonts/" !default;
       content: "\e6e8";
     }
   }
```

Comment:
Below are the new-new icons, but review above—up through "desktop"—for those added in the last update, which should also be new since you last imported to the Demo product.